react/tic-tac-toe/src/app/screens/GameHistory/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+
+import styles from './styles.module.scss';
+
+import MatchesService from '~services/MatchesService';
+
+const Spinner = require('react-spinkit');
+
+class GameHistory extends Component {
+  state = {
+    isLoading: true,
+    history: [],
+    error: ''
+  };
+
+  getData() {
+    MatchesService.getMatches()
+      .then(response => {
+        if (response.ok) {
+          this.setState({
+            isLoading: false,
+            history: response.data
```

Comment:
And this could be moved from component local state to redux store, what do you think?

Reply:
Yes. I change this logic on the next branches. This screen handle states with recompose.